packages/cli/src/common-options.ts

```diff
@@ -11,7 +11,7 @@ export function populateWithCommonOptions(command: Command) {
 		.addOption(
 			new Option(
 				"-n, --network <string>",
-				"The network name of the SEDA chain, can be mainnet, testnet, devnet, planet or other",
+				"The network name of the SEDA chain, can be mainnet, testnet, devnet, planet or other (default: testnet)",
```

Comment:
commander already displays the default:testnet:

```
-n, --network <string>  The network name of the SEDA chain, can be mainnet, testnet, devnet, planet or other (default: "testnet", env: SEDA_NETWORK)
```

Reply:
Thanks!  🤦